src/backend/distributed/commands/dependencies.c

```diff
@@ -390,6 +390,12 @@ ReplicateAllObjectsToNodeCommandList(const char *nodeName, int nodePort)
 	ObjectAddress *dependency = NULL;
 	foreach_ptr(dependency, dependencies)
 	{
+		if (IsObjectAddressOwnedByExtension(dependency, NULL))
+		{
+			/* we expect extension-owned objects to already exist on worker nodes */
```

Comment:
This function is actually called when we distribute all objects to a new worker I believe. The following comment would make more sense in that situation:

```suggestion
			/* we expect extension-owned objects to be created as a result of the extension being created */
```

Reply:
Done.